.gitignore

```diff
@@ -0,0 +1,3 @@
+.env
+
```

Comment:
Our convention here is to have these ignore files listed one after the other and then add a new line to the end of the file. Could you remove this space and add a newline to the end of the file?